otx/cli/utils/tests.py

```diff
@@ -742,11 +762,27 @@ def otx_build_auto_config(root, otx_dir: str, args: Dict[str, str]):
     workspace_root = os.path.join(root, "otx-workspace")
     command_line = ["otx", "build", "--workspace-root", workspace_root]
 
-    for option, val in args.items():
+    for option, value in args.items():
         if option in ["--train-data-roots", "--val-data-roots"]:
-            command_line.extend([option, f"{os.path.join(otx_dir, val)}"])
+            command_line.extend([option, f"{os.path.join(otx_dir, value)}"])
         elif option in ["--task"]:
             command_line.extend([option, args[option]])
     check_run(command_line)
 
     shutil.rmtree(workspace_root)
+
+
+def otx_train_auto_config(root, otx_dir: str, args: Dict[str, str]):
+    work_dir = os.path.join(root, "otx-workspace")
+    command_line = ["otx", "train"]
+
+    for option, value in args.items():
+        if option == "template":
+            command_line.extend([args[option]])
+        elif option in ["--train-data-roots", "--val-data-roots"]:
+            command_line.extend([option, f"{os.path.join(otx_dir, value)}"])
+    command_line.extend(["--save-model-to", f"{work_dir}"])
+    command_line.extend(["--work-dir", f"{work_dir}"])
+    command_line.extend(args["train_params"])
+    check_run(command_line)
+    shutil.rmtree(work_dir)
```

Comment:
How about using TemporaryDirectory in [tempfile](https://docs.python.org/3/library/tempfile.html) instead deleting manually?

Reply:
As far as I know, this uses a TemporaryDirectory. 
@sungmanc Any other reason for this erasing code?

Reply:
Removed this. https://github.com/openvinotoolkit/training_extensions/pull/1641/commits/3fcf067f3ae5f70826b0df66bdb702db0a04c358

Reply:
I couldn't remember the reason, maybe I added this line when we don't allow the duplicate build. So, it can be removed.